x-pack/platform/plugins/shared/onechat/public/application/components/tools/execute/test_tools.tsx

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiButton,
+  EuiFieldText,
+  EuiFieldNumber,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutHeader,
+  EuiForm,
+  EuiFormRow,
+  EuiSpacer,
+  EuiTitle,
+} from '@elastic/eui';
+import React, { useState } from 'react';
+import { useForm, FormProvider, Controller } from 'react-hook-form';
+import type { ToolDefinition } from '@kbn/onechat-common';
+import { i18n } from '@kbn/i18n';
+import { CodeEditor } from '@kbn/code-editor';
+import { useExecuteTool } from '../../../hooks/tools/use_execute_tools';
+import type { ExecuteToolResponse } from '../../../../../common/http_api/tools';
+
+interface OnechatTestToolFlyoutProps {
```

Comment:
Let's keep props consistent with component name

```suggestion
interface OnechatTestFlyout {
```